src/node.ts

```diff
@@ -133,7 +138,7 @@ function importResolve(input: string, cacheRoot: string, packageRoot: string): P
     return typeof specifier !== "string" || // AST node?
       isNodeBuiltin(specifier) || // node built-in, e.g., "node:fs" or "fs"
       isPathImport(specifier) || // relative path, e.g., ./foo.js
-      /^\w+:/.test(specifier) || // windows file path, https: URL, etc.
+      /^\0?[\w-]+:/.test(specifier) || // windows file path, https: URL, \x00node-resove:, etc.
```

Comment:
For reasons I really don’t understand, the node-resolve plugin tries to resolve things like `"\x00node-resolve:empty.js"`. If we don’t return null here, we’ll try to resolve the id as a node package, which obviously will fail. I think this expanded pattern will match the weird internal names used by node-resolve now. 🤷